notebooks/notebooks/iclr2022/estimate-gpu-days-adv.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    # - On cp505, 3 A6000 victimplay workers generated 250k rows in 9h30m and 7
</pre><p>Reminds of me of mathcounts problems haha.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>